django/test/utils.py

```diff
@@ -841,6 +844,30 @@ def disable(self):
         setattr(Options, 'default_apps', self.old_apps)
 
 
+class time_keeper:
```

Comment:
Why not `TimeKeeper`? 

We instantiate this later: `self.time_keeper = time_keeper(self.timing)`, which could equally be `self.time_keeper = TimeKeeper(self.timing)`

Reply:
I thought it should be snake_case since users might want to use the class themselves to time their own test runner's operations or time something else entirely i.e it's a documented interface. Reading [PEP8](https://www.python.org/dev/peps/pep-0008/#class-names) again though does show it should be camelCase instead since it isn't going to be primarily used as a callable